pkg/printers/internalversion/printers.go

```diff
@@ -1538,6 +1540,9 @@ func getNodeInternalIP(node *api.Node) string {
 
 // findNodeRoles returns the roles of a given node.
 // The roles are determined by looking for:
+// * a role.node.kubernetes.io/<role>="" label
+// * a kubelet.kubernetes.io/role="<role>" label
```

Comment:
I wonder if `role.kubelet.kubernetes.io/<role>=""` is more consistent with the node approach?

Reply:
Agree, I have updated the PR.    Please review it again, Thanks!